src/CookieBanner.elm

```diff
@@ -78,3 +80,26 @@ innerContainer =
     batch
         [ width (pct 100)
         ]
+
+
+cookieBtnStyle : Style
+cookieBtnStyle =
+    batch
+        [ pseudoElement "after"
+            [ backgroundImage (url "images/arrow--white.svg")
```

Comment:
@aaaaargZombies I think this needs to be "/images/arrow--white.svg"

I know is merged :grimacing: